sdk/cosmos/azure-spring-data-cosmos-core/pom.xml

```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
-      <version>4.4.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
```

Comment:
@kushagraThapar Was this package also released along with the `4.4.0-beta.1` azure-cosmos release?

Reply:
@samvaity - no, this was not released. Only azure-cosmos module was released.